app/views/comments/_comment.html.erb

```diff
@@ -3,18 +3,27 @@
     <tr>
       <th scope="col">Commenter</th>
       <th scope="col">Comment</th>
+      <th scope="col">Image</th>
+      <th scope="col">commenter Name</th>
     </tr>
   </thead>
   <tbody>
     <tr>
       <td><%= comment.commenter %></td>
       <td><%= comment.body %></td>
-      <td align="center">
+      <td>
```

Comment:
this part being duplicated many times. Do you think we should move it to a partial?

Reply:
updated